src/Service/Rekognition/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## NOT RELEASED
 
+## 0.1.5
```

Comment:
We should also add that we added operations `getCelebrityInfo()` and `recognizeCelebrities()`